### ISSUE TITLE
Can not find the model

### ISSUE THREAD

**Issue by jjelosua** [evanwill/argonautproject#1]
Hi,

I have tried to follow your instructions but nevertheless I sill get these errors:

Did your instructions for the tika server work for you anytime?

[INFO ] 2020-04-30 16:59:35,962 org.apache.tika.parser.ner.NamedEntityParser - going to load, instantiate and bind the instance of org.apache.tika.parser.ner.opennlp.OpenNLPNERecogniser
[WARN ] 2020-04-30 16:59:35,963 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-location.bin using class loader
[INFO ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - LOCATION NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-organization.bin using class loader
[INFO ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - ORGANIZATION NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-date.bin using class loader
[INFO ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - DATE NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-money.bin using class loader
[INFO ] 2020-04-30 16:59:35,964 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - MONEY NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-person.bin using class loader
[INFO ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - PERSON NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-percentage.bin using class loader
[INFO ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - PERCENT NER : Available for service ? false
[WARN ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - Couldn't find model from org/apache/tika/parser/ner/opennlp/ner-time.bin using class loader
[INFO ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.opennlp.OpenNLPNameFinder - TIME NER : Available for service ? false
[INFO ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.NamedEntityParser - org.apache.tika.parser.ner.opennlp.OpenNLPNERecogniser is available ? false
[INFO ] 2020-04-30 16:59:35,965 org.apache.tika.parser.ner.NamedEntityParser - going to load, instantiate and bind the instance of org.apache.tika.parser.ner.regex.RegexNERecogniser
[INFO ] 2020-04-30 16:59:35,970 org.apache.tika.parser.ner.NamedEntityParser - org.apache.tika.parser.ner.regex.RegexNERecogniser is available ? false
[INFO ] 2020-04-30 16:59:35,983 org.apache.tika.parser.ner.NamedEntityParser - Number of NERecognisers in chain 0
java -Dlog4j.configuration=file:config/tika/log4j.properties -classpath /Users/xxx/code/11p/gitlab/FARO/models:/tmp/tika-server.jar org.apache.tika.server.TikaServerCli -h 0.0.0.0

**Comment by evanwill** [comment]
@jjelosua sorry, I did not notice your Issue until just now. The instructions in this repo are 4 years old, so probably something is out of date and may not work anymore. The versions linked to are all old. It also looks like it is trying to load more models than the three that I set up in those instructions as well. 

If you successfully downloaded the models, be sure to check that their location is correct. In this basic set up everything needs to be at the correct file path. It assumes you have a user `tika-user`, with a directory `tika` where you put the `.jar` file and the directory `tika-ner-resources` where you downloaded the models (and put them in the correct directory structure). 
That is it assumes you have folder "/home/tika-user/tika/" with the .jar, and folder "/home/tika-user/tika/tika-ner-resources/org/apache/tika/parser/ner/opennlp/" with the models. If you put the files you downloaded somewhere else, you will need to adjust the command `java -classpath` to match your actual file locations.

Good luck.
